rootdir/vendor/etc/libnfc-nci.conf

```diff
@@ -1,17 +1,42 @@
+###################### Start of libnfc-brcm.conf #######################
+
 ###############################################################################
 # Application options
-NFC_DEBUG_ENABLED=0x00
+APPL_TRACE_LEVEL=0xFF
+PROTOCOL_TRACE_LEVEL=0xFFFFFFFF
+
+NFC_DEBUG_ENABLED=0x01
```

Comment:
NONONONONONONO! Set it to zero!

Reply:
That's what we get for blatantly copying things from stock _days_ ago without ever reading through :grin: